__tests__/date-time-offset/1.0/date-time-offset.test.js

```diff
@@ -0,0 +1,238 @@
+import { addMinutes } from 'date-fns';
+import dateTimeOffset from '../../../functions/date-time-offset/1.0';
+
+describe('Date Time Offset', () => {
+  test('return date time for 1 second added to a new javascript date for UTC+1 zone', async () => {
+    const currentDate = true;
+    const customStartDate = null;
+    const offsetType = 'ss';
+    const offset = '1';
+    const resultType = 'DT';
+    const TimeZoneOffset = '60';
+    const checkDate = addMinutes(new Date(), parseInt(TimeZoneOffset, 10));
```

Comment:
This is done to keep the function test readable. 
These settings would change per test, so putting them outside the test function does not help readability.
Putting them in an object was a great idea and has been done.

Reply:
I think it has become less readable due to he amount of data, you can think of making shared variables where you can just change a few properties of

Reply:
I disagree. For each test, only a few of them change, and thus would require figuring out the other variables yourself. Having the variables in each test would allow the user to fully understand each test instead of having to piece the different variables together.

Reply:
I agree with Bob (and uncle Bob.. ;) ) here. As another dev trying to understand what is going on I actually find that it is rather helpful that I can see all the variables in one glance in every test instead of having to scroll back. It speeds up the process and allows me to easily change variables if needed.